README.md

```diff
@@ -8,7 +8,11 @@ Curious about the philosopy being used to create the index? See [these notes](ht
 
 ## Running locally
 
-You should be able to run the site locally just by opening `/public/index.html`
+To run the site locally, just start a simple http server pointing at the "public" folder:
+
+```bash
+npx http-server public -o -p 9999
```

Comment:
Just opening the file directly in the browser didn't work because the latest CORS policy doesn't allow an HTML file to pull in any local `js` files at all :expressionless:.  [MDN recommended](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server) running a simple server via `npx http-server public -o -p 9999` and that did the trick, so I took the liberty of adding that to README.